hardhat.config.ts

```diff
@@ -10,6 +12,11 @@ import { HardhatUserConfig } from "hardhat/config"
 import "solidity-coverage"
 import "./mocha-test"
 
+dotenv.config()
+const optimismGoerliUrl = process.env.OPTIMISM_GOERLI_URL || "http://localhost:8545"
+const mnemonic = process.env.MNEMONIC || "test"
+const optimisticEtherscanAPIKey = process.env.OPSCAN_API_KEY || "test"
+
```

Comment:
I prefer to change this part and make it more universal like in this [file](https://github.com/eludius18/ZK--SNARKS_ZoKrates_Loan/blob/main/hardhat.config.ts) with all referenced to dotenv and ready to be deployed in multiple Blockchains